test/util/baseUtils.test.js

```diff
@@ -0,0 +1,10 @@
+const { checkSupportedResource } = require('../../src/util/baseUtils');
+
+describe('Testing base utility functions', () => {
```

Comment:
based on my catch in the other comment, maybe we should add a test to make sure that `checkSupportedResource(undefined)` also throws an error here